frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -14,7 +14,7 @@
         @click='addSelectedMention(index)'
       )
         avatar(:src='user.picture' size='xs')
-        .c-username {{user.displayName}}
+        .c-username {{user.username}}
```

Comment:
If they have a display name (and only if a display name is set), can you add it in a grey font in parenthesis after the username?

E.g. so that the mentions looks something like this:

```
[IMG] greg
[IMG] snow (Snowteamer)
[IMG] alex (Alex Jin)
```